### ISSUE TITLE
Blank windows if network is not available when the application starts

### ISSUE THREAD

**Issue by hexion** [Thomas101/wmail#418]
I noticed this while travelling and I used the airport free wifi network, These networks typically require the user to go onto a web page to log in, or even click on a "I agree" type button. Before that happens, no outgoing connections will succeed.

In this scenario and by the time outgoing connections are finally allowed, WMail will display a blank page for each of the email accounts. The only way to progress is by restarting WMail.

* Which version of WMail are you using? [e.g. 1.3.1]
2.0.5, also reproduced in 2.0.0

* Which Operating System are you using? [e.g. Windows 10, Mac OSX 10, Ubuntu 14.04]
Ubuntu 14.0.4

**Comment by Thomas101** [comment]
Thanks, I don't think I handle that case at the moment. I handle the one where you're completely offline. I'll need to check to see if I can capture it and show a different screen. In the meantime, if you hit up against this again you can use `Ctrl+R` to reload the WMail window once you are connected

**Comment by hexion** [comment]
A polling solution such as pinging gmail's server every minute or so may solve the problem. 

But it's not really a big deal, in my opinion it is more desirable to have a certain delay while the network is unresponsive (whether it's disconnected or pending authorization on a public network) than having the application fail at start and never recover from that. But it's just my opinion, the Ctrl+R workaround works fine.

However, if the current approach is kept it would be good to add a status icon that signifies that something has gone wrong. Currently, if the application is set to start minimized, the user cannot tell the difference between having no new emails or if something has gone wrong.

**Comment by Thomas101** [comment]
I agree would definitely be good to report some kind of status on this and checking gmail is probably the way to go
